packages/core/src/components/time-picker-rework/time-picker-rework.tsx

```diff
@@ -0,0 +1,328 @@
+        (item) => !item.hidden
+      );
+    }
+
+    return (
+      <Host>
+        <ix-date-time-card
+          standaloneAppearance={this.standaloneAppearance}
+          corners={this.corners}
+        >
+          <div class="header" slot="header">
+            <ix-typography variant="default-title">Time</ix-typography>
+          </div>
+          <div class="clock">
+            {timepickerInformation.map((descriptor, index: number) => (
+              <div class="flex">
+                <div class={{ columns: true, hidden: descriptor.hidden }}>
+                  <ix-icon-button
+                    size="16"
+                    onClick={() =>
+                      (this._time = this._time.add(1, descriptor.unit))
+                    }
+                    ghost
+                    icon="chevron-up"
+                    variant="primary"
+                    class="arrows"
+                  ></ix-icon-button>
+
+                  <input
+                    class="form-control"
+                    name={descriptor.unit}
+                    type="number"
+                    placeholder={descriptor.placeholder}
+                    min="0"
+                    max={dayjs().endOf('day').get(descriptor.unit)}
+                    value={this._formattedTime[descriptor.unit]}
+                    onChange={(e) => {
+                      let inputElement = e.target as HTMLInputElement;
+                      inputElement.value = this.timeUpdate(
+                        descriptor.unit,
+                        +inputElement.value
+                      ).toString();
+                    }}
+                  ></input>
+
+                  <ix-icon-button
+                    size="16"
+                    onClick={() =>
+                      (this._time = this._time.subtract(1, descriptor.unit))
+                    }
+                    ghost
+                    icon="chevron-down"
+                    variant="primary"
+                    class="arrows"
+                  ></ix-icon-button>
+                </div>
+
+                {index !== timepickerInformation.length - 1 && (
+                  <div
+                    class={{
+                      'column-seperator': true,
+                      hidden: descriptor.hidden,
+                    }}
+                  >
+                    :
+                  </div>
+                )}
+              </div>
+            ))}
+
+            <div
+              class={{
+                columns: true,
+                'default-space': true,
+                hidden: this._timeRef === undefined,
+              }}
+            >
+              <ix-icon-button
+                size="16"
+                onClick={() => this.changeTimeReference()}
+                ghost
+                icon="chevron-up"
+                variant="primary"
+                class="arrows"
+              ></ix-icon-button>
+              <div class="time-reference">{this._timeRef}</div>
```

Comment:
Invalid time/format result in error message between time reference toggle buttons.

We should hide the AM/PM field an log a message to the console (console.error) that there is an issue regarding the given time and format.

```
 <ix-time-picker-rework
          time="10:11:12"
          format="hh:mm:ss A"
/>
```

Result: 
![image](https://github.com/siemens/ix/assets/2144313/a0552032-07e8-42d7-bd88-8486598b8c3d)